src/tests/tracing/eventpipe/simpleruntimeeventvalidation/simpleruntimeeventvalidation.cs

```diff
@@ -2,15 +2,17 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 
 using System;
+using System.Collections.Generic;
```

Comment:
Are any of the changes in this file needed? It looks like this is just refactoring pre-existing test code unrelated to the new feature. Lets revert the changes in this file if they aren't needed.

Reply:
This file used to contain the validation of AllocationSampled events that is now in allocationsampling.
I'm reverting it back to its original state.
✅ done